rbac/api/tasks.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Celery tasks."""
+from __future__ import absolute_import, unicode_literals
+
+from celery import shared_task
+
+from api.cross_access.util import check_cross_request_expiry
+
+
+@shared_task
+def cross_account_cleanup():
+    """Celery task to mark expired cross-account requests."""
+    check_cross_request_expiry()
```

Comment:
Came across a situation locally testing, where things appeared to not be working when I ran this. It turned out to be that it was failing silently because I didn't setup the principal (I didn't update the car to "approved" via the API, I did it directly in postgres, so it never provisioned it). 

Since it runs in a background process, the exceptions never get raised/logged, it just silently stops. Maybe adding a try/except here at least to log any exceptions would be helpful? And/or a metric (here or in the actual method) so we don't get a false positive that it's running, but something is actually failing to delete them? I was confused because my cars were not updating to expired.

Reply:
That's a great idea, I'll work on getting that added in!